tests/e2e-test.ts

```diff
@@ -50,6 +50,12 @@ export async function e2eTest({
   const [recipientSenderConnection, senderRecipientConnection] = await makeConnection(recipientAgent, senderAgent)
   expect(recipientSenderConnection).toBeConnectedWith(senderRecipientConnection)
 
+  // Create default link secret
+  await recipientAgent.modules.anoncreds.createLinkSecret({
```

Comment:
I think in this high-level e2e test it could be nice to call createLinkSecret() with default parameters just to check that it is set as default and everything works in a transparent way (I guess most users will only register a single link secret and won't care about its particular id).

Reply:
Updated!